### ISSUE TITLE
Unable to import data before 05-Feb-2019 into Postgresql

### ISSUE THREAD

**Issue by noobvie** [mimblewimble/grin-explorer#72]
Hello Hendi,

It's me again, finally I could get most data of blockchain, just one small issue remaining, I can't get data before 05-Feb-2019. Not sure if it's due to hard-fork during that period or python or postgresql limitation. 
I also discovered that Grinmint got same issue like me, you could see the examples here:
https://scan.grin.money/block/000000bbf9aacf955891b3e5703d06d1649921ca6216dd15b22b31595c4fe0ac
   Block 29,143
https://grinmint.com/block/000002a3eed065f456ce9f31377389ee85f605cec9d3c4824f972ddbe618efa9
  Block 29,141

When we click on the inputs, it shown 404 not found error.
Thanks for your help.

**Comment by hendi** [comment]
What happened when you ran the full import? Didnot go all the way through to block 0? If not, any errors? When did it stop?

Can you check directly in the database if all blocks are there? (I.e. is it an issue during your import, or with display)

**Comment by noobvie** [comment]
Hi Hendi,
I forgot to capture the screen after the import done, after checking the height before Feb, they did not exist in database. I tried to import again but it did not detect old heights before Feb as well. 
So I worked around by changing the scenario, I would not run the node to complete the sync first. Now I will setup everything for explorer run well, then running the node will be the last step, I scheduled the python import command every minute in crontab.
Currently, the postgresql could import gradually while the node is syncing. And I could see my genesis block here  https://scan.grin.money/block/0

Finally, I think we could move the grin node part to the end of Readme file, so new explorer would not face my same issue.
Thank you.
